### ISSUE TITLE
No bower.json file?

### ISSUE THREAD

**Issue by girishramnani** [puneet24/realtime_notification#1]

**Comment by puneet24** [comment]
Yeah I do remember about that but solving first core issues and then  I will add all the necessary things. Feel free to chime in between if you want. :+1:
